README.md

```diff
@@ -68,10 +68,10 @@ That said, if you have questions, reach out to us
 [announcement]: https://cncf.io/news/announcement/2015/07/new-cloud-native-computing-foundation-drive-alignment-among-container
 [Borg]: https://research.google.com/pubs/pub43438.html
 [CNCF]: https://www.cncf.io/about
-[communication]: https://git.k8s.io/community/communication
-[community repository]: https://git.k8s.io/community
+[communication]: https://github.com/kubernetes/kubernetes/blob/master/SUPPORT.md
```

Comment:
i don't think the communication and support pages have the exact same purpose?
is there a tracking issue for this change?

@kubernetes/sig-contributor-experience-pr-reviews

Reply:
i just found related pages, maybe not suitable

Reply:
Just to add on, the communication link lists all communication platforms that people can use to interact. That page also lists support platforms.